src/components/v5/frame/PageLayout/PageLayout.tsx

```diff
@@ -44,6 +53,16 @@ const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({
     };
   }, []);
 
+  useLayoutEffect(() => {
```

Comment:
Please move those changes to `src/components/frame/Extensions/layouts/SharedLayout.tsx`

Reply:
Fixed